src/pages/dashboard.tsx

```diff
@@ -54,7 +80,134 @@ let DashboardPage = ({baseUrl}) => {
+                    <Paragraph>
+                        If you account is already in our system, you can search for it below, or <a
+                        onClick={() => setNewUserSearch(newUserSearch === 'search' ? 'create' : 'search')}>
+                        you can add your new account and we'll start tracking your stats right away</a>
+                    </Paragraph>
+
+                    <GamerSearchInput size={'lg'}
+                                      baseUrl={baseUrl}
+                                      mode={'condensed'}
+                                      onGamerClick={newUserGamerSelect}
+                                      focus={false}/>
+                </Show>
+
+                <Show show={newUserSearch === 'create'}>
+                    <Paragraph>
+                        You can add your gamer details, or <a
+                        onClick={() => setNewUserSearch(newUserSearch === 'search' ? 'create' : 'search')}>
+                        if your account is already in our system, you can search for it here
+                    </a>
+                    </Paragraph>
+
+                    <GamerAdd recaptchaSiteKey={process.env.NEXT_PUBLIC_WARZONE_RECAPTCHA_SITE_KEY}/>
+                </Show>
+            </>
+        );
+    }
+
+
+
+    function newUserGamerSelect(gamer) {
+        if (gamer) {
+            const {username, platform} = gamer;
+
+            let userIsOkay = confirm(`select OK to make ${username} (${GamerService.getPlatformObjByID(platform).name}) your main`);
+
+            if (userIsOkay) {
+                GamerRelationshipService.createGamerRelationship({
+                    user_id: user.user_id,
+                    username: username,
+                    platform: platform,
+                    type: 'self'
+                }).then((gamerRelationship) => {
+                    console.log(gamerRelationship);
+                    getGamerRelationships();
+                }).catch((error) => {
+                    console.log(error);
+                });
+            }
+        }
+    }
+
+
+
+    function getData() {
+        console.log('get data');
+        getGamerRelationships().finally(() => {
+            setLoading(false);
+            setDataHasLoaded(true);
+        });
+    }
+
+
+
+    function getGamerRelationships() {
+        return new Promise((resolve, reject) => {
+
+            GamerRelationshipService.queryGamerRelationships({
+                user_id: user.user_id
+            }).then((_gamerRelationships) => {
+                console.log('_gamerRelationships', _gamerRelationships);
+                setGamerRelationships(_gamerRelationships);
+                resolve();
+            }).catch((error) => {
+                console.error(error);
+                reject(error);
+            });
+        });
+    }
 };
 
 
-export default DashboardPage;
\ No newline at end of file
+export default dynamic(() => Promise.resolve(DashboardPage), {ssr: false});
```

Comment:
Disabled SSR

Reply:
Because it's only for a logged-in experience right?